### ISSUE TITLE
[Feature request] GPU external fan headers

### ISSUE THREAD

**Issue by rainmasak** [Rem0o/FanControl.Releases#307]
ASUS GPUs have external fan headers attached directly to GPU itself (meant to control case fans). These controls however are not visible in the application. If possible, please detect them.

**Comment by Rem0o** [comment]
" please detect them"

If only it was that easy. Interop documentation with these kinds of things are not public.

**Comment by rainmasak** [comment]
Yeah, feared as much :) might also need some propietary ASUS drivers, as this functionality is only available in ASUS Gpu Tweak (crappy) software so far.

**Comment by Rem0o** [comment]
If you ever got hold of such doc/driver, would be a good plugin idea: https://github.com/Rem0o/FanControl.Releases/wiki/Plugins
